site/src/components/Form/Form.tsx

```diff
@@ -121,6 +122,7 @@ export const FormSection: FC<
         >
           {title}
           {alpha && <AlphaBadge />}
+          {deprecated && <DeprecatedBadge />}
```

Comment:
Can you drop a screencap with what this looks like?

Reply:
![image](https://github.com/coder/coder/assets/11241812/93a4f280-1fd2-40df-8cfd-df17ee45d305)